src/backend/utils/html/sanitize.js

```diff
@@ -41,10 +41,12 @@ module.exports = function (dirty) {
       'tr',
       'ul',
     ],
+    allowedSchemesByTag: { img: ['http', 'https', 'data'] },
     allowedAttributes: {
       iframe: ['src'],
       img: ['src'],
       a: ['href'],
+      blockquote: ['cite'],
```

Comment:
Why is this here?

Reply:
Just wanted to add an additional test, hm, it does look irrelevant though in hindsight. I'll remove it from this PR.

Reply:
It's a great change, just do it in its own issue/pr.